test/controllers/employments/UpdateEmploymentControllerSpec.scala

```diff
@@ -82,7 +82,8 @@ class UpdateEmploymentControllerSpec extends BaseSpec {
     inject[UpdateEmploymentCheckYourAnswersView],
     inject[ConfirmationView],
     inject[ErrorPagesHandler],
-    mockRepository
+    mockRepository,
+    mockEmpIdCheck
```

Comment:
Should not we have the case for emp id check not found, and the backend services are not called for all the controller specs